admin/server-timing.php

```diff
@@ -168,3 +212,61 @@ class="large-text code"
 	</p>
 	<?php
 }
+
+/**
+ * Renders a checkbox for enabling output buffering for Server-Timing.
+ *
+ * @since n.e.x.t
+ */
+function perflab_render_server_timing_page_output_buffer_checkbox() {
+	$slug           = 'output_buffering';
+	$field_id       = "server_timing_{$slug}";
+	$field_name     = PERFLAB_SERVER_TIMING_SETTING . '[' . $slug . ']';
+	$description_id = "{$field_id}_description";
+	$has_filter     = has_filter( 'perflab_server_timing_use_output_buffer' );
+	$is_enabled     = perflab_server_timing_use_output_buffer();
+
+	?>
+	<div class="output-buffering-field">
+		<input
+			type="checkbox"
+			id="<?php echo esc_attr( $field_id ); ?>"
+			name="<?php echo esc_attr( $field_name ); ?>"
+			aria-describedby="<?php echo esc_attr( $description_id ); ?>"
+			<?php disabled( $has_filter ); ?>
+			<?php checked( $is_enabled ); ?>
+		>
+		<label for="<?php echo esc_attr( $field_id ); ?>">
+			<?php esc_html_e( 'Enable output buffering of template rendering.', 'performance-lab' ); ?>
+		</label>
+		<p id="<?php echo esc_attr( $description_id ); ?>" class="description">
+			<?php if ( $has_filter ) : ?>
+				<?php if ( $is_enabled ) : ?>
+					<?php
+					echo wp_kses(
+						sprintf(
+							/* translators: %s: perflab_server_timing_use_output_buffer */
+							__( 'Output buffering has been forcibly enabled via the %s filter.', 'performance-lab' ),
+							'<code>perflab_server_timing_use_output_buffer</code>'
+						),
+						array( 'code' => array() )
+					);
+					?>
+				<?php else : ?>
+					<?php
+					echo wp_kses(
+						sprintf(
+							/* translators: %s: perflab_server_timing_use_output_buffer */
+							__( 'Output buffering has been forcibly disabled via the %s filter.', 'performance-lab' ),
+							'<code>perflab_server_timing_use_output_buffer</code>'
+						),
+						array( 'code' => array() )
+					);
+					?>
+				<?php endif; ?>
+			<?php endif; ?>
+			<?php esc_html_e( 'This is needed to capture metrics after headers have been sent and while the template is being rendered. Note that output buffering may possibly cause an increase in TTFB if the response would be flushed multiple times.', 'performance-lab' ); ?>
```

Comment:
Maybe we should change the "This is needed" to another wording that works regardless of whether the setting is force-controlled or not? When the filter is force-disabled, it doesn't work well with the previous sentence. It sounds like disabling output buffering is needed...
<img width="820" alt="Screenshot 2023-08-17 at 12 35 20 PM" src="https://github.com/WordPress/performance/assets/3531426/956d7fbc-4023-4310-8f5b-faa2854f6bd1">

Reply:
How about:

```suggestion
			<?php esc_html_e( 'Output buffering is needed to capture metrics after headers have been sent and while the template is being rendered. Note that output buffering may possibly cause an increase in TTFB if the response would be flushed multiple times.', 'performance-lab' ); ?>
```

Reply:
SGTM!